src/pspm_split_sessions.m

```diff
@@ -166,8 +166,11 @@
         missing = round(missing*missingsr); % convert epochs in sec to datapoints
     end
     indx = zeros(1,round(missingsr * ininfos.duration)); % indx should be a one-dimensional array?
-    indx(missing(:, 1)) = 1;
-    indx(missing(:, 2)+1) = -1;
+    % allow splitting empty missing epochs

```

Comment:
I agree the old code always take effects on the missing epochs, which is not expected. It should be applied to missing epochs when users define missing epoch files.